### DISCUSSION TITLE
Informacja o ukryciu oferty w zasobie /sale/offer-events

### DISCUSSION THREAD

**Discussion by dawidczech355a** [allegro/allegro-api#6417] [Suggestions - rejected]
Zauważyliśmy, że zasób `/sale/offer-events` nie informuje o ukryciu oferty przez panel WWW.

Kiedy dokonamy jakiejkolwiek zmiany oferty przez panel Allegro, odpytując zasób `/sale/offer-events` otrzymujemy informację na temat przeprowadzonych modyfikacji ofert, jednak kiedy ukryjemy ofertę przez panel WWW, żadna informacja nie jest przekazywana przez wspomniany endpoint. 

Chcielibyśmy zmienić nasz wewnętrzny status oferty na podstawie ukrycia jej w panelu Allegro.
Sugeruję dodanie nowego typu, np. `{ OFFER_HIDDEN }`.

**Comment by MartaNowaczyk** [comment]
Zaktualizowaliśmy status sugestii jako "odrzucona", ponieważ nie planujemy sugerowanych zmian.
